video/out/vo_kitty.c

```diff
@@ -113,6 +120,35 @@ static struct sigaction saved_sigaction = {0};
 static bool resized;
 #endif
 
+static inline void write_bstr_passthrough(struct priv *p, bstr bs)
+{
+    write_bstr(p->dcs_prefix);
+    write_bstr(bs);
+    write_bstr(p->dcs_suffix);
```

Comment:
This was my initial idea, but instead of storing `dcs_prefix` and `dcs_suffix`. We could store kitty commands templates in priv, prepared in init. I know you said it wouldn't look nice, but would really be much different?

Reply:
not really sure but i feel like doing `*_passthrough(...)` is better than having
```c
if (passthrough) {
  sprintf(priv->some_esc, "...", ...);
} else {
  priv->some_esc = "...";
}
```

if you really want me to do that i have no trouble changing it

Reply:
ok, I don't mind either way.